project/Dependencies.scala

```diff
@@ -9,14 +9,15 @@ object Dependencies {
   val Scala212 = "2.12.17"
   val ScalaVersions = Seq(Scala213, Scala212)
 
-  val AkkaVersionInDocs = "2.7"
+  val AkkaVersionInDocs = "2.7.0"
```

Comment:
This was 2.7. The purpose is to not point doc URLs to an exact patch release, but latest 2.7.x